wifiphisher/common/interfaces.py

```diff
@@ -383,6 +383,11 @@ def is_interface_valid(self, interface_name, mode=None):
                 raise InvalidInterfaceError(interface_name, mode)
             elif mode == "AP" and not interface_adapter.has_ap_mode:
                 raise InvalidInterfaceError(interface_name, mode)
+
+            # raise an error if interface is already in the _active set
+            if interface_name in self._active:
+                raise InvalidInterfaceError(interface_name)
```

Comment:
@anakin1028 Can you add the exception to the `docstring` of the method. Also can you add a warning about the fact that a valid interface name will be added to the active list.